$OpenBSD: patch-doc_sam_1,v 1.2 2015/04/18 20:35:25 schwarze Exp $
Ignore .TF macro that is broken even with groff; use default indentation.
Delete unused .EX and .EE macros.
--- doc/sam.1.orig	Sat Apr 18 15:40:50 2015
+++ doc/sam.1	Sat Apr 18 15:53:01 2015
@@ -15,19 +15,7 @@
 .de CW
 .ft B
 ..
-.\" This is gross but it avoids relying on internal implementation details
-.\" of the -man macros.
 .de TF
-.br
-.IP "" \w'\fB\\$1\ \ \fP'u
-.PD0
-..
-.de EX
-.CW
-.nf
-..
-.de EE
-.fi
 ..
 .\" delete above this point if your system has F, L, FR, LR, CW and TF macros
 .ds a \fR*\ \fP
